### PR TITLE
Improve heading hierarchy on sitemap page.

### PR DESCRIPTION
**Pull request by NickColley**
Raised by AJ on Cross GOV Slack.

### CODE REVIEW

**Review by domoscargin — APPROVED** [review]
Nice catch! This looks good to me.

Just need another approval before merging.

**Review by owenatgov — APPROVED** [review]
Looks dandy to me

**Pull request merged**
